pycross/private/package_repo.bzl

```diff
@@ -201,8 +206,9 @@ def _package_repo_impl(rctx):
     _generate_lock_bzl(rctx, lock_json_path, lock_bzl_path)
 
     for pin, pin_target in lock["pins"].items():
+        pin = pin.replace("-", "_")
```

Comment:
This is a breaking change, not sure how this could be handled. Just wanted to demonstrate that it is possible.